src/bindings/python/src/openvino/runtime/opset9/ops.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Factory functions for all ngraph ops."""
+from functools import partial
+from typing import Optional
+
+import numpy as np
+from openvino.runtime import Node
+from openvino.runtime.opset_utils import _get_node_factory
+from openvino.runtime.utils.decorators import nameable_op
+from openvino.runtime.utils.types import (
+    NodeInput,
+    as_nodes,
+    as_node
+)
+
+_get_node_factory_opset9 = partial(_get_node_factory, "opset9")
+
+
+# -------------------------------------------- ops ------------------------------------------------
+
+
+@nameable_op
+def eye(
+        num_rows: NodeInput,
+        num_columns: NodeInput,
+        diagonal_index: NodeInput,
+        output_type: str,
+        batch_shape: Optional[NodeInput] = None,
+        name: Optional[str] = None,
+) -> Node:
```

Comment:
I would imagine it as:
```suggestion
@nameable_op
def eye(
        num_rows: NodeInput,
        num_columns: NodeInput,
        diagonal_index: NodeInput,
        batch_shape: Optional[NodeInput] = None,
        output_type: str = "f32",
        name: Optional[str] = None,
) -> Node:
```
It would require changes in cpp part but at the same time it will be more aligned with frameworks and result in better style.

Also please take a look at examples from numpy:
https://numpy.org/devdocs/reference/generated/numpy.eye.html

Reply:
If we decide to introduce default value for "f32", I will introduce it in a separate PR to review just related changes.